src/views/workflow-summary-tab/workflow-summary-tab-json-view/workflow-summary-tab-json-view.styles.ts

```diff
@@ -6,7 +6,7 @@ import type {
 const cssStylesObj = {
   jsonViewContainer: (theme) => ({
     padding: theme.sizing.scale600,
-    backgroundColor: theme.colors.backgroundSecondary,
```

Comment:
please correct the tertiaryBackground color in the theme and use it. Keeping both web and design inconsistent would make moving forward much harder